### PR TITLE
Add FAQ to Huggingface page

### PR DESCRIPTION
**Pull request by danielholanda**
Closes https://github.com/groq/mlagility/issues/134

**Task:** Adds FAQ to Huggingface page

### CODE REVIEW

File: trackers/huggingface/app.py [hunk 1]
```diff
@@ -21,10 +21,44 @@ def add_faq() -> None:
...
+            "their specific problems, and then take the path of least resistance into "
+            "production. A market for turnkey solutions, starting with a model as input and "
+            "provision a cost- and latency-effective acceleration solution, often in the cloud, "
+            "as output, has emerged."
+        ),
+    )
+    faq.add_section(
+        "What are the current key limitations of MLAgility?",
+        (
+            "Groq's latency is computed using GroqModel.estimate_latency(), which takes"
```
Comment (kovtcharov) [inline trackers/huggingface/app.py:59]: Is this really a limitation of the MLAgility tool per se? Seems more of a limitation of the Groq toolflow instead, no? Are there other limitations of MLAgility that would be worth mentioning here that might be more relevant to the broader community?

**Review by kovtcharov — APPROVED** [review]

File: trackers/huggingface/app.py [hunk 2]
```diff
@@ -21,10 +21,44 @@ def add_faq() -> None:
...
+            "as-is, nor manually optimize their models by hand-writing kernels for "
+            "hardware accelerators. MLAgility addresses the disconnect between MLPerf and "
+            "mass adoption customers by examining a larger corpus of models, while "
+            "disallowing hand-written kernels."
+        ),
+    )
+    faq.add_section(
+        "Who runs MLAgility?",
+        (
+            "MLAgility is lead and maintained by a consortium of industry leaders and innovators."
```
Comment (jeremyfowers) [inline trackers/huggingface/app.py:41]: Make this more specific (currently there's no mention of Groq)

File: trackers/huggingface/app.py (hunk 1 above)
Comment (jeremyfowers) [reply trackers/huggingface/app.py:59]: Off the top of my head:

1. Missing important classes of models (LLMs, GNNs, DLRMs, etc.)
2. Missing batching and sweeps over input sizes
5. Only three devices are represented (missing the full product lines, eg T4, A10, H100, etc. for nvidia)
6. Only three vendors are represented (missing ARM, AMD, etc.)
7. Only 1 runtime SW is represented per device (missing ORT and PyTorch for CUDA, PyTorch for x86, etc.)

File: trackers/huggingface/app.py [hunk 3]
```diff
@@ -21,10 +21,44 @@ def add_faq() -> None:
     """
     faq = Collapsable()
```
Comment (jeremyfowers) [inline trackers/huggingface/app.py:22]: Add a section for the experimental setup. What specific devices were used, in what cloud, with what software version numbers? What is the benchmarking methodology (can link to the benchit user guide). Mention that benchit was used to collect the benchmarks.

**Review by jeremyfowers — CHANGES_REQUESTED** [review]

File: trackers/huggingface/app.py (hunk 3 above)
Comment (danielholanda) [reply trackers/huggingface/app.py:22]: Added methodology sections. We really have to be careful here to always update that section.

File: trackers/huggingface/app.py [hunk 4]
```diff
@@ -21,10 +21,92 @@ def add_faq() -> None:
...
     faq.add_section(
-        "Why is this so empty?",
+        "How is MLAgility different from MLPerf?",
         (
-            "Because the FAQ of huggingface website still needs to be written. "
-            "We don't use the same FAQ as in our internal dashboard."
+            "Deep learning pioneers have been judging their progress with the Machine Learning "
```
Comment (jeremyfowers) [inline trackers/huggingface/app.py:26]: If all these line breaks are annoying to maintain you should feel free to use a pylint-except to disable the line-length linting rule.

File: trackers/huggingface/app.py [hunk 5]
```diff
@@ -21,10 +21,92 @@ def add_faq() -> None:
...
+            "of the MLAgility Github Repository. You can learn more about it "
+            '<a href="https://github.com/groq/mlagility">here</a>.'
+        ),
+    )
+    faq.add_section(
+        "What is the experimental setup for each of the devices?",
+        [
+            "<b>x86</b>: Intel(R) Xeon(R) X40 CPU @ 2.00GHz on Google Cloud (custom: n2, 80 vCPU, 64.00 GiB) and OnnxRuntime version 1.14.0.",
+            "<b>nvidia</b>: NVIDIA A100 40GB on Google Cloud (a2-highgpu-1g) and TensorRT version 22.12-py3.",
+            "<b>groq</b>: GroqChip 1 on GroqFlow version 2.5.2 and GroqWare™ Suite version 0.9.2.",
```
Comment (jeremyfowers) [inline trackers/huggingface/app.py:62]:
```suggestion
            "<b>groq</b>: GroqChip 1 on GroqFlow version 3.0.2 test package and GroqWare™ Suite version 0.9.2.",
```

That's what's in setup.py. Unless you were using older groqflow?

File: trackers/huggingface/app.py [hunk 6]
```diff
@@ -21,10 +21,92 @@ def add_faq() -> None:
...
+            "their specific problems, and then take the path of least resistance into "
+            "production. A market for turnkey solutions, starting with a model as input and "
+            "provision a cost- and latency-effective acceleration solution, often in the cloud, "
+            "as output, has emerged."
+        ),
+    )
+    faq.add_section(
+        "Which tool was used to generate those results?",
+        (
+            "All MLAgility results have been generated using the <b>benchit</b> tool, which is part "
```
Comment (jeremyfowers) [inline trackers/huggingface/app.py:52]:
```suggestion
            "All MLAgility results have been generated using the <b>benchit</b> tool v1.0.0, which is part "
```

File: trackers/huggingface/app.py (hunk 3 above)
Comment (jeremyfowers) [reply trackers/huggingface/app.py:22]: Yeahhh keeping all of that up to date will be effort-intensive if we do it manually, and super bad if we ever forget. 

Filed https://github.com/groq/mlagility/issues/189 to address this later

**Review by jeremyfowers — APPROVED** [review]

**Pull request merged**